buildsystem/dependencies.gradle

```diff
@@ -52,8 +52,8 @@ ext.versions = [
 ]
 
 ext.libraries = [
-        store                      : "org.mobilenativefoundation.store:store4:$versions.store",
-        filesystem                 : "org.mobilenativefoundation.store:filesystem4:$versions.store",
+        store                      : "com.dropbox.mobile.store:store4:$versions.store",
```

Comment:
why did this go backwards?

Reply:
It's temporary until we release to new coordinates. CI tests were failing https://github.com/MobileNativeFoundation/Store/actions/runs/3570145503/jobs/6000839230